android/java/org/chromium/chrome/browser/BraveRewardsHelper.java

```diff
@@ -428,13 +428,15 @@ public static String getCurrentYear(Resources resources) {
         return Integer.toString(currentTime.get(Calendar.YEAR));
     }
 
-  public static Tab currentActiveChromeTabbedActivityTab() {
-      ChromeTabbedActivity activity = BraveRewardsHelper.getChromeTabbedActivity();
-      if (activity == null || activity.getTabModelSelector() == null) {
-          return null;
-      }
-      return activity.getActivityTab();
-  }
+    public static Tab currentActiveChromeTabbedActivityTab() {
+        ChromeTabbedActivity activity = BraveRewardsHelper.getChromeTabbedActivity();
+        if (activity == null
+                || !activity.areTabModelsInitialized()
+                || activity.getTabModelSelector() == null) {
```

Comment:
could you please also change `getTabModelSelector`(it's deprecated) to `getTabModelSelectorSupplier` as

Reply:
thanks, changed at https://github.com/brave/brave-core/pull/26724/commits/3ba588610aaee273642f29625cb7750a85b372b0